### PR TITLE
Add datatypes for Rust

### PR DESCRIPTION
**Pull request by cubetastic33**
Add datatypes to the Rust language definition and add datatype definitions in the DataType directory.

### CODE REVIEW

File: data/Language/Rust.yaml [hunk 1]
```diff
@@ -20,7 +20,19 @@ delimiters:
...
+  - bool_rust
+  - char_rust
+  - i8_rust
+  - u8_rust
+  - i16_rust
+  - u16_rust
+  - i32_rust
+  - u32_rust
+  - i64_rust
+  - i64_rust
```
Comment (Naveenaidu) [inline data/Language/Rust.yaml]: Please remove it. It's repeated twice

File: data/Language/Rust.yaml [hunk 2]
```diff
@@ -20,7 +20,19 @@ delimiters:
...
+  - u16_rust
+  - u32_rust
+  - u64_rust
+  - i8_rust
+  - i16_rust
+  - i32_rust
+  - i64_rust
+  - f32_rust
+  - f64_rust
+  - string_rust
```
[the remark this answers is not in the document]
Comment (cubetastic33) [reply data/Language/Rust.yaml]: Thanks, I've updated it

**Comment by cubetastic33** [comment]
Oops, I think I missed two datatypes (`i128` & `u128`). One moment, I'll push another commit

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/pull/64/commits/d7c3948c2a5ec0f52489a2a5f940edd3dede6cf7

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**